charts/topolvm/CHANGELOG.md

```diff
@@ -7,8 +7,22 @@ This file itself is based on [Keep a CHANGELOG](https://keepachangelog.com/en/0.
 
 ## [Unreleased]
 
+## [10.0.0] - 2022-11-04
 ### Added
 - feat: add `args` for `controller`, `node`, `scheduler`, `lvmd` to helm chart (#576)
+- helm: Add resource template to csi-registrar and liveness-probe containers (#579)
+
+### Changed
+- helm: Update templates for Container specific resources (#582)
+  - **BREAKING**: Changed parameters to set resources in `values.yaml` file.
+
+### Removed
+- github/workflows: Use output parameter instead of set-output command (#581)
```

Comment:
IMO, this is not "Removed" since no behavior changes happen.

Reply:
@daichimukai Thank you. I moved this line to `Changed` section.